EternalModLoader/Mod.cs

```diff
@@ -14,6 +14,11 @@ public class Mod
         /// Mod file bytes
         /// </summary>
         public byte[] FileBytes;
+        
+        /// <summary>
+        /// True if the mod file is a .blang json
+        /// </summary>
+        public bool isBlangJson;
```

Comment:
Please follow C# coding conventions. See: https://docs.microsoft.com/es-es/dotnet/csharp/programming-guide/inside-a-program/coding-conventions

Please use PascalCase for properties.